OctoKitTests/TestHelper.swift

```diff
@@ -22,8 +22,15 @@ internal class Helper {
     }
 
     internal class func codableFromFile<T>(_ name: String, type: T.Type) -> T where T: Codable {
+        var path: String
         let bundle = Bundle(for: self)
-        let path = bundle.path(forResource: name, ofType: "json")!
+        if let bundlePath = bundle.path(forResource: name, ofType: "json") {
```

Comment:
Why was this changed? The remaining test are untouched.

Reply:
@pietbrauer I added some reasoning in the PR description. Here is the snippet:

> * I made changes to the `codableFromFile` method and the reasoning was so that I could run tests from the commandline `swift test` but there are sources that are copied over using bundle resources but Swift PM doesn't support it.
> * I am happy to remove those changes if we don't want to support SPM going forward

Reply:
Ah, I see. Sorry for not remembering it.